.travis.yml

```diff
@@ -36,7 +36,7 @@ env:
 
 install:
   - git clone https://github.com/openresty/test-nginx.git test-nginx
-  - sudo luarocks install lua-resty-http 0.14-0
+  - sudo luarocks install lua-resty-http
   - sudo luarocks install lua-typeof
```

Comment:
BWT, I think we can use `sudo make dev` to install dependencies via Luarocks.

We can do this in a new PR later.